### ISSUE TITLE
SyntaxError in main line 218

### ISSUE THREAD

**Issue by jaraco** [rspeer/ordered-set#96]
Attempting to use ordered set from main, it fails with a SyntaxError and fails to install.

```
 ~ 🐚 pip-run git+https://github.com/rspeer/ordered-set -- -c "import ordered_set"
  error: subprocess-exited-with-error
  
  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [21 lines of output]
      Traceback (most recent call last):
        File "/Users/jaraco/.local/pip-run/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 353, in <module>
          main()
          ~~~~^^
        File "/Users/jaraco/.local/pip-run/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 335, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
                                   ~~~~^^^^^^^^^^^^^^^^^^^^^^^^
        File "/Users/jaraco/.local/pip-run/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 118, in get_requires_for_build_wheel
          return hook(config_settings)
        File "/private/var/folders/f2/2plv6q2n7l932m2x004jlw340000gn/T/pip-build-env-ys6j54g6/overlay/lib/python3.13/site-packages/flit_core/buildapi.py", line 32, in get_requires_for_build_wheel
          docstring, version = get_docstring_and_version_via_ast(module)
                               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^
        File "/private/var/folders/f2/2plv6q2n7l932m2x004jlw340000gn/T/pip-build-env-ys6j54g6/overlay/lib/python3.13/site-packages/flit_core/common.py", line 149, in get_docstring_and_version_via_ast
          node = ast.parse(f.read())
        File "/opt/python/lib/python3.13/ast.py", line 54, in parse
          return compile(source, filename, mode, flags,
                         _feature_version=feature_version, optimize=optimize)
        File "<unknown>", line 218
          raise ValueError(f"Argument needs to be an iterable, got {type(sequence}")
                                                                                 ^
      SyntaxError: closing parenthesis '}' does not match opening parenthesis '('
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
```

**Comment by jaraco** [comment]
Oh, god. This issue was reported and patched in #90, but the patch has gone unreviewed. :(

**Comment by jondo** [comment]
This is now fixed.
